backend/load_balancer.conf

```diff
@@ -0,0 +1,13 @@
+http {
```

Comment:
The network diagram that describes this configuration is here: https://www.notion.so/nextchaptersoftware/Network-Diagram-badd0c06413d4976a16690773309f1a3